accelerometer/summariseEpoch.py

```diff
@@ -18,7 +18,7 @@ def getActivitySummary(epochFile, nonWearFile, summary,
     epochPeriod=30, stationaryStd=13, minNonWearDuration=60,
     mgCutPointMVPA=100, mgCutPointVPA=425,
     activityModel="activityModels/walmsley-jan21.tar",
-    intensityDistribution=False, useRecommendedImputation=True,
+    intensityDistribution=False, intensityDistributionVals = "accelerometer/intensityVals.txt", useRecommendedImputation=True,
```

Comment:
Unexpected spaces around keyword / parameter equals (E251)
https://www.flake8rules.com/rules/E251.html
Default is `accelerometer/intensityDistributionVals.txt`?